source/ranges.tex

```diff
@@ -1316,6 +1319,10 @@
   template<size_t N, class I, class S, subrange_kind K>
     requires (N < 2)
   constexpr auto get(const subrange<I, S, K>& r);
+
+  template<size_t N, class I, class S, subrange_kind K>
+    requires (N < 2)
+    constexpr auto get(subrange<I, S, K>&& r);
```

Comment:
```suggestion
  constexpr auto get(subrange<I, S, K>&& r);
```

Reply:
The indentation here is correct; the previous `get` is wrong.